app.js

```diff
@@ -0,0 +1,78 @@
+function getHistory(){
```

Comment:
Feedback not addressed?

Reply:
I have changed the Names and I will update it soon !